src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/DoubleUtil.cs

```diff
@@ -57,12 +59,7 @@ internal static class DoubleUtil
         /// <param name="value2"> The second double to compare. </param>
         public static bool AreClose(double value1, double value2)
         {
-            //in case they are Infinities (then epsilon check does not work)
-            if(value1 == value2) return true;
-            // This computes (|value1-value2| / (|value1| + |value2| + 10.0)) < DBL_EPSILON
-            double eps = (Math.Abs(value1) + Math.Abs(value2) + 10.0) * DBL_EPSILON;
-            double delta = value1 - value2;
-            return(-eps < delta) && (eps > delta);
+            return Math.Abs(value1 - value2) < 10.0 * DBL_EPSILON;
```

Comment:
The bad news is I tested to replace Math.Abs, but the result will be changed. 

In my other PR, I have tried to do it that is  similar your code, but I failed. See https://github.com/dotnet/wpf/pull/5571

Reply:
Will the result be changed essentially or are these just minor details? As described in the comment, the method is not expected to deliver exact results.

Reply:
@deeprobin Just minor details.